core-lib/src/main/java/org/idpass/smartscanner/lib/nfc/details/NFCDocumentTag.kt

```diff
@@ -230,6 +260,10 @@ class NFCDocumentTag(val readDG2: Boolean = true) {
                     } else {
                         passportCallback.onGeneralException(cardServiceException)
                     }
+                    // Capture card exceptions and Send to Sentry
+                    if (captureLog) {
+                        Sentry.captureException(cardServiceException)
```

Comment:
I just want to make sure that `cardServiceException` prints out a readable string message rather than an object address here...

Reply:
@typelogic here is a sample of `cardServiceException` in Sentry, will this be alright? or do I need to send `localizedMessage` /`message` of exception instead?

![image](https://user-images.githubusercontent.com/11598375/126607367-5aea7a21-15ff-4400-9edd-fcf9e142fc53.png)
![image](https://user-images.githubusercontent.com/11598375/126607331-3c05fc50-f6f8-4983-9bf2-31155f0a597b.png)

Reply:
I tested `println(cardServiceException)` and the printed string is discernable enough. So, indeed, this is fine.